phpstan.conditional.config.php

```diff
@@ -0,0 +1,15 @@
+<?php
+declare(strict_types = 1);
+
+$config = array();
+
+if (version_compare(PHP_VERSION, '8.0', '>=')) {
```

Comment:
PHP 8.0's signature has changed for most CURL functions, and in this case, when PHPStan is run on PHP 8 it complains about this `CurlHandle` class not being allowed. If I add `CurlHandle` as a return type to the docblock, then PHP < 8 complains that the class doesn't exist. It may be possible to use PHPStan's php 8 stubs project to fix this, but this conditional ignore works well for now.